### ISSUE TITLE
console: pretty print objects

### ISSUE THREAD

**Issue by legendecas** [Rokid/ShadowNode#259]
Right now `console.log` did not print objects in an information rich way. 

```
-> console.log({})
[object Object]
```

A possible pretty print of object could be adopted.

```
-> console.log({ foo: 'bar' })
{
  foo: "bar"
}
```

**Comment by yorkie** [comment]
A pretty implementation of `inspect()` is needed for this issue.

**Comment by legendecas** [comment]
#140 was merged for a performance issue, yet it introduced an flaws that an object cannot be pretty printed if it is the first few arguments.

It would be great if both performance and pretty print could be considered in `console.log`

**Comment by legendecas** [comment]
resolved by #311
